### DISCUSSION TITLE
Welcome to the community!

### DISCUSSION THREAD

**Discussion by omeraplak** [refinedev/refine#1055] [General]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together 🚀 

To get started, comment below with an introduction of yourself and tell us about what you are planning to build with refine. Also, feel free to join us on [Discord](https://discord.gg/refine)!

**Comment by aboucodeur** [comment]
Hello , refine i'am fullstack developper i like new technologie you can add postgresql but it's the one of the most popular database today in world for startup .

**Comment by sorooshh** [comment]
hi everyone i am new to programing  i want to use the  refine app but  i have a PROBLEM, I want to create refine in VSC using terminal command " npm create refine-app@latest " but the  fallowing answer shows in terminal :( npm : The term 'npm' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or 
if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ npm create refine-app@latest
+ ~~~
    + CategoryInfo          : ObjectNotFound: (npm:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException )
also i cant open the app in the home page (( Your Refine application will be accessible at http://localhost:****/: ))
because : (localhost refused to connect.)!
what do i do

**Reply by sorooshh** [reply]
please answer

**Reply by BatuhanW** [reply]
Hey @sorooshh as a general suggestion, please consider reading error messages. They are obviously telling you the problem. And if you don't understand the problem by readin the message, you can also Google it or ask to ChatGPT.

From what I understand from the message, it seems you haven't installed `npm` locally. Please do so. You can use `nvm` to install it.

**Comment by dlarsson** [comment]
All the discord links seem to be invalid. Tried the one here, on the home page and on Twitter, but no go.

**Reply by aliemir** [reply]
Hey @dlarsson, sorry for the trouble! Our invite links have expired somehow and now it's fixed and working! Join our community at https://discord.gg/refine 🚀

**Comment by vitalijalbu** [comment]
Hi @omeraplak , could you share some Real use cases on your website about companies using refine? And how they use it... ?
Thanks. 

PS. I spoked with the refine memeber but he didn't replied yet...
